packages/dev/core/src/Rendering/fluidRenderer/fluidRenderer.ts

```diff
@@ -16,6 +16,8 @@ import { FluidRenderingTargetRenderer } from "./fluidRenderingTargetRenderer";
 import { FluidRenderingObjectCustomParticles } from "./fluidRenderingObjectCustomParticles";
 import { FluidRenderingDepthTextureCopy } from "./fluidRenderingDepthTextureCopy";
 
+import "@babylonjs/core/Engines/Extensions/engine.transformFeedback";

```

Comment:
I don't think it's the responsibility of the fluid renderer to import this file...

This file is needed by the (GPU) particle system, not directly by the fluid renderer, which can work even without a particle system.

Also, what about WebGPU if it is imported here? This file is specific to WebGL2.

Reply:
ohhhhh good catch I ll move it into the gpu particle system. About WebGPU, this is probably were some of the ESM work done by @RaananW will need clever trickery :-)

We tend to import the required extensions for other features.

Reply:
ESM will require actively initializing specific modules, which will include adding the needed extensions. i'm finalizing the API in the next few weeks